classification/support/view_cell_interactively.m

```diff
@@ -260,7 +260,7 @@ function show_other_cells(show)
         
         for m = 1:num_other_cells
             set(other_cell_handles(m,1), 'Visible', vis_val);
-            set(other_cell_handles(m,2), 'Visible', vis_val);
+%             set(other_cell_handles(m,2), 'Visible', vis_val);
```

Comment:
Yes, I also thought of the lazy eval approach. However, there's still the issue that 1000 `text`s on the small movie is actually useless.

I think the right approach is to use tooltips here as well, or programmatically determine when the plot view is over a particular ROI of the movie, and apply `text`s over local regions. The latter approach would be significantly more work.